server/taskflows/hpccloud/taskflow/pyfr/__init__.py

```diff
@@ -498,7 +498,21 @@ def upload_output(task, _, cluster, job, *args, **kwargs):
     # step.
     file_names = [f['name'] for f in solution_files]
     file_names.sort()
-    vtu_file = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
+    if len(file_names) > 1:
+        vtu_fileFirst = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
+        # find where to put the ...*...
+        head = tail = size = len(vtu_fileFirst)
+        for fileName in file_names[1:]:
+            vtu_name = '%s.vtu' % fileName.rsplit('.', 1)[0]
+            for i in range(size):
+                if vtu_fileFirst[i] != vtu_name[i] and head > i:
+                    head = i # not included in the cut
+                if vtu_fileFirst[-i] != vtu_name[-i] and tail >= i:
+                    tail = i - 1
+
+        vtu_file = '%s*%s' % (vtu_fileFirst[:head], vtu_fileFirst[-tail:])
+    else:
+        vtu_file = '%s.vtu' % file_names[0].rsplit('.', 1)[0]
     task.taskflow.set_metadata('vtuFile', vtu_file)
```

Comment:
A general comment this seems pretty complex, can't we just pass *.vtu to paraview?